opentelemetry-api/tests/correlationcontext/test_correlation_context_propagation.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2020, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import typing
+import unittest
+
+from opentelemetry import correlationcontext
+from opentelemetry.context import get_current
+from opentelemetry.correlationcontext.propagation import (
+    CorrelationContextPropagator,
+)
+
+
+def get_as_list(
+    dict_object: typing.Dict[str, typing.List[str]], key: str
+) -> typing.List[str]:
+    value = dict_object.get(key)
+    return value if value is not None else []
```

Comment:
I know this function is defined multiple times and you just took the definition from somewhere else, in any case, it can be written as follows:

```suggestion
    return dict_object.get(key, [])
```

Reply:
Fixed